servicetalk-dns-discovery-netty/gradle/spotbugs/test-exclusions.xml

```diff
@@ -43,4 +43,8 @@
     <Source name="~.*Test\.java"/>
     <Bug pattern="THROWS_METHOD_THROWS_CLAUSE_THROWABLE"/>
   </Match>
+  <Match>
+    <Source name="~.*Test\.java"/>
+    <Bug pattern="ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD"/>
```

Comment:
If you follow the above recommendation, spotbugs won't rise this error and you won't need `@SuppressWarnings("PMD.MutableStaticState")`